src/jdk.jdi/share/classes/com/sun/tools/jdi/EventSetImpl.java

```diff
@@ -496,6 +505,19 @@ String eventName() {
         }
     }
 
+/*[IF CRIU_SUPPORT]*/
+    class VMRestoreEventImpl extends ThreadedEventImpl implements VMRestoreEvent {
+
+        VMRestoreEventImpl(JDWP.Event.Composite.Events.VMRestore evt) {
+            super(evt, evt.requestID, evt.thread);
+        }
+
+        String eventName() {
```

Comment:
This is missing the `@Override` annotation.

Reply:
Added